app/src/main/java/com/courseproject/tindar/entity/Account.java

```diff
@@ -0,0 +1,44 @@
+package com.courseproject.tindar.entity;
```

Comment:
Can we have a folder name entities as plural since we will put multiple entities in this folder?

Reply:
done